configure.ac

```diff
@@ -19,14 +19,16 @@ AC_PREREQ([2.68])
 #   fourier_major_version += 1;
 #   fourier_minor_version = 0;
 #
-m4_define([fourier_major_version], [0.4])
-m4_define([fourier_minor_version], [5])
+m4_define([fourier_major_version], [0])
```

Comment:
not sure to understand how the version number is now handled in the autotools chain?

Reply:
if keeping gimp2 fourier here in this project, and moving gimp3 to a new project folder this is easy.
For the gimp3 fourier, clone gimp2 project, merge the gimp2.99 branch into main, and that becomes the main branch (similar to what I did with gimp3-fix-ca).
For the gimp2 fourier, delete the gimp2.99 branches (so the main branch stays as-is).

If you want to have both gimp2 and gimp3 in the same project, the versioning will need to be duplicated, sort of like what I did with [libuninameslist english and french](https://github.com/fontforge/libuninameslist/blob/master/configure.ac) lines 7..27.
configure.ac, Makefile.am will get more complicated.
It then lets you have both gimp2 and gimp3 in one project, but I also think making it more complicated tends to keep new users from adding patches if the code gets complicated too.

Take a couple days to think which direction you want to go from here, both directions are possible, but each has it's pros and cons.

Reply:
I think the version number should reflect the core fourier plugin functionalities, but not on the GIMP version. So as the core codebase has not been modified, I think the plugin version should remain 0.4.5.

About the repository strategy, I was thinking to merge the gimp2.99 branch no later than the official GIMP 3.0 version, and at as soon as the CI will be able to build Windows binaries. It should be the case on the next GIMP development release (that should be a 3.0-rc1 version), mingw64 packages are often quickly available. At this stage the CI will not be able to build the deb package, but it seems to be OK to me: as GIMP3 is not available in those repos, there will be only power users that will need it, and power users will be able to compile it themselves.

Reply:
Ok - sounds like the strategy is to keep everything as one project.